nvflare/private/fed/server/server_engine.py

```diff
@@ -176,6 +176,8 @@ def heartbeat_to_parent(self):
             except BaseException:
                 # The parent process can not be reached. Terminate the child process.
                 break
+        # delay some time for the wrap up process before the child process self terminate.
```

Comment:
why 30 seconds ? why not 15 or 46 seconds ?

Reply:
We can't have delay too long. Otherwise, it may cause a confusion that a ghost child process may still be running when the job is done. If too short, it may result to a controller not be able to finish the abort wrap up process. I think 30 seconds for that is reasonable.